.circleci/config.yml

```diff
@@ -0,0 +1,190 @@
+version: 2.1
+
+parameters:
+  snapshot:
+    type: boolean
+    default: true
+  release:
+    type: boolean
+    default: false
```

Comment:
Do we need 2 parameters? Something is either a release or snapshot, it can never be both or neither?

Reply:
We do need both. You need to tell CCI which workflow to kick off and that if done with params.
`--data '{"branch":"'${BRANCH}'","parameters":{"release": true, "snapshot": false}}' \`

Reply:
Mhmmm, why can't we get away with 1 parameter "release" which defaults to false. The API command is the only time we want to set it to true? The build-and-test workflow runs when release = false.

Reply:
Done. Quit bugging me omfg 🤣